main/src/main/java/cgeo/geocaching/ImageDetailActivity.java

```diff
@@ -0,0 +1,147 @@
+package cgeo.geocaching;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.Bundle;
+import android.view.GestureDetector;
+import android.view.MenuItem;
+import android.view.MotionEvent;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import com.igreenwood.loupe.Loupe;
+
+public class ImageDetailActivity extends AppCompatActivity {
```

Comment:
Don't create a new activity.

* If you want to jump to a new activity for image display, then use  `cgeo.geocaching.ImageViewActivity`. tryx use method `openImageView` from it to open new activity
* If instead you want to make the image zoomable inside the Edit-Activity, take a look at method `cgeo.geocaching.utils.ImageUtils.createZoomableImageView`